bin/create_test_data.js

```diff
@@ -8,114 +8,82 @@
 
 'use strict';
 
-var uuid = require('uuid');
-var program = require('commander');
-var pg = require('pg');
-
-var createHash = require('crypto').createHash;
 var config = require('../server/config');
-var models = require('../server/models');
-var user = models.user;
 var log = require('../server/logger')('bin.createTestData');
+var user = require('../server/models/user');
+var visitsController = require('../server/controllers/visits');
+var testUrls = require('../config/test-urls');
 
-var defaultCount = 25;
-
-// see bottom for initialization
-
-var dbParams = {
-  user: config.get('db.postgres.user'),
-  password: config.get('db.postgres.password'),
-  host: config.get('db.postgres.host'),
-  port: config.get('db.postgres.port'),
-  database: config.get('db.postgres.database'),
-  ssl: config.get('db.postgres.ssl')
-};
-
-function done() {
-  log.verbose('should now exit; done invoked');
-  process.exit();
+if (!config.get('testUser.enabled')) {
+  throw new Error('To create test data, you must set testUser.enabled in the config.');
 }
 
-function createTestUser(recordCount, cb) {
-  recordCount = recordCount || defaultCount;
-
-  var bulkData = [];
-
-  // we'll use this date as a starting point for generating records. Each successive
-  // record will be a number of seconds in the future.
-  var historyDate = new Date('2015-01-01T21:26:23.795Z');
-
+function createTestUser(cb) {
   var fakeUser = {
     id: config.get('testUser.id'),
     email: config.get('testUser.email'),
     oauthToken: 'fakeOauthToken'
   };
   log.verbose('fakeUser is ' + JSON.stringify(fakeUser));
-
-  function generateTestRecord(n) {
-    var visitedAt = new Date(historyDate.getTime() + (1000 * n)).toJSON();
-    var title = 'Title for Page Number ' + n;
-    var url = 'https://www.record' + n + '.com/whatever';
-    var urlHash = createHash('sha1').update(url).digest('hex').toString();
-    // in order: visitId, visitedAt, fxaId, rawUrl, url, urlHash, title
-    var output = [uuid.v4(), visitedAt, fakeUser.id, url, url, urlHash, title];
-    log.trace('generateTestRecord output is: ' + JSON.stringify(output));
-    return output;
-  }
-
-  // TODO when we start generating large test data sets, stream it in
-  for (var j = 0; j < recordCount; j++) {
-    log.trace('pushing a record onto bulkData');
-    bulkData.push(generateTestRecord(j));
-  }
-
   user.create(fakeUser.id, fakeUser.email, fakeUser.oauthToken, function(err) {
-    if (err) {
-      return cb && cb(err);
-    }
-    pg.connect(dbParams, function(err, client, done) {
-      function donezo(str, err) {
-        if (str) { log.warn(str); }
-        if (err) { log.trace(err); }
-        done();
-        pg.end();
-        cb && cb(err);
-      }
+    return cb && cb(err);
+  });
+}
 
-      if (err) {
-        return donezo('failed to connect to database', err);
+function createTestData(cb) {
+  var userId = config.get('testUser.id');
```

Comment:
ESLint says that we aren't using `userId` anywhere. Not sure if we want to remove this, or possibly change the `generateTestRequest()` `auth.credentials` value below on line 42 to use the value (instead of requerying the config for each loop iteration.